android/app/src/main/res/layout/fragment_filter.xml

```diff
@@ -0,0 +1,153 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@drawable/bg_app"
+    android:padding="@dimen/default_margin_24">
+
+    <View
+        android:id="@+id/vFilterSendFromBg"
+        android:layout_width="match_parent"
+        android:layout_height="0dp"
+        android:layout_marginTop="@dimen/default_margin_40"
+        android:background="@drawable/bg_filter"
+        app:layout_constraintBottom_toBottomOf="@id/tilFilterSendFromTemplate"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <TextView
+        style="Widget.Template.TextView.Bold"
+        android:id="@+id/tvFilterSendFrom"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/default_margin_16"
+        android:layout_marginTop="@dimen/default_margin_16"
+        android:layout_marginEnd="@dimen/default_margin_16"
+        android:text="@string/filter_send_from"
+        android:textAllCaps="true"
+        android:textColor="@color/white"
+        android:textSize="@dimen/dimen_display_text_18"
+        app:layout_constraintTop_toTopOf="@id/vFilterSendFromBg" />
+
+    <com.google.android.material.textfield.TextInputLayout
+        style="@style/AppTextInputLayout"
+        android:id="@+id/tilFilterSendFromProviderName"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/default_margin_16"
+        android:layout_marginTop="@dimen/default_margin_8"
+        android:layout_marginEnd="@dimen/default_margin_16"
```

Comment:
We could combine `android:layout_marginStart` and `android:layout_marginEnd` by `android:layout_marginHorizontal`. Please update in the next upcoming PRs 🙏